source/process/casestudy.rst

```diff
@@ -92,4 +92,16 @@ If you're completing the questionnaire by email, please send a copy of the quest
 
 Thank you so much for being part of our community!
 
+Written Case Study Lay Out
+----------------------------
+
+Follow these steps to write a case study that is clear and concise:
+
+1. Title: Focus on highlighting the most compelling accomplishment.
```

Comment:
Usually bolding shouldn't be used unless it is a field name or clickable control - Jeff once directed me to the bolding guidelines in the "Documentation guidelines"